src/components/game/methods/createData.js

```diff
@@ -0,0 +1,28 @@
+const createData = (boardSize, cardType) => {
+  let arr = [];
+  let maxId = 100;
+  const a = boardSize/2;
+
+  for (let i = 0; i < a; i++) {
+    const el = {
+      text: `${i+1}`,
+      isOpened: false,
+      isGuessed: false,
+      isNotGuessed: false,
+      pause: false,
+      imageSrc: `./images/${cardType}/${i + 1}.${cardType === 'cards' ? 'png' : 'jpg'}`
+    };
+    arr = [ ...arr, el, el]; 
```

Comment:
arr - const, поэтому тут пуш.
```suggestion
    arr.push( el ); 
```